source/manual/alerts/search-benchmarking.html.md

```diff
@@ -8,14 +8,9 @@ last_reviewed_on: 2017-10-02
 review_in: 3 months
 ---
 
-Indicates that the Jenkins [search_benchmark healthcheck job]
-(https://deploy.publishing.service.gov.uk/job/search_benchmark/) has failed.
+Indicates that the Jenkins [search_benchmark healthcheck job] (https://deploy.publishing.service.gov.uk/job/search_benchmark/) has failed.
 
 The healthcheck sends queries to the search API and compares the results to the
-expected pages.
+expected pages. It will only fail if something actually goes wrong - for example, if a request to the search API fails. Check the output of the job on the relevant environment for more information.
```

Comment:
This is essentially correct although I've made it swallow the exception if it's a server error.

Reply:
So will this error ever appear?

Reply:
@tomnatt as discussed it'll just be a lot less likely to appear.

Reply:
So page is still relevant (as per Chris's original question). I will merge.